contracts/contract/src/lib.rs

```diff
@@ -125,14 +144,14 @@ impl NewsStorage {
     #[payable]
     pub fn nft_mint(
         &mut self,
-        index: usize,
+        index: usize
     ) -> Token {
         assert!(index < self.news.len());
-        assert!(self.news[index].like>=100, "There are not enough likes to publish this news");
+        //assert!(self.news[index].like>=100, "There are not enough likes to publish this news");
```

Comment:
@dariacherniak magic numbers is considered bad practice, named const is always better choice as it gives descriptive name so develper understand what this constant suppose to be.

p. s. please also make it eq 3 for testing/demo purpose